drivers/dma/dma_pl330.h

```diff
@@ -56,7 +56,7 @@
 #define DMAC_PL330_DBGINST0	0xd08
 #define DMAC_PL330_DBGINST1	0xd0c
 
-#define DMA_TIMEOUT_US		10
+#define DMA_TIMEOUT_US		100000
```

Comment:
Nit: Adding a comment to why this timeout is this value could be useful, especially since it's changing by 4 orders of magnitude here. Can be done as a follow-up diff though, doesn't need to hold this up.

Reply:
@olofj : 10us was too less for 128K bytes data transfer and we saw the DMA channel status appearing busy on FPGA/emulation setup, so set the max value 100000 to accommodate all possible data transfer sizes.  Timeout of 100000us may never hit.

Reply:
Yeah, seems very high but timeouts should be very rare and critical issues when they happen.

 Having that in the header file as a brief comment to why the number is the value it is would be useful, otherwise someone needs to go find the pull request and see the above reply when they're trying to figure why it is that way.

Reply:
@olof:  Agree, Let me put the comment explaining the reason. Thanks.